jmxtrans-output/jmxtrans-output-elastic/src/test/java/com/googlecode/jmxtrans/model/output/elastic/ElasticWriterTests.java

```diff
@@ -208,13 +234,27 @@ private ElasticWriter createElasticWriter() {
 		Map<String,Object> settings = new HashMap<String,Object>();
 
 		String connectionUrl = "http://localhost";
+		ElasticWriter writer;
+
+		try {
+			writer = new ElasticWriter(typenames, true, PREFIX, true, connectionUrl, null, null, null, null, settings);
+		} catch (IOException e) {
+			throw new RuntimeException("Unexpected failure to create elastic writer for test", e);
```

Comment:
if I propagate this IOException, i will have unhand exception warning at line: 78 and 81. Will it fine ?